gmso/tests/test_forcefield_xml.py

```diff
@@ -200,3 +200,7 @@ def test_potential_types_by_expression(self, named_groups_ff):
         assert len(dihedral_types_grouped_by_expression['0.5*z*(r - r_eq)**2']) == 2
         assert len(improper_types_gropued_by_expression['0.5*z*(r - r_eq)**2']) == 1
 
+    def test_forcefield_missing_atom_types(self):
+        with pytest.raises(MissingAtomTypesError):
+            ff = ForceField(get_path(filename=get_path('ff_missing_atom_types.xml')))
```

Comment:
can we add an exact same line but with `strict=False` just to make sure the forcefield will load.

Reply:
I have added the test in 0a89771.